packages/psnotify.vm/tools/chocolateyuninstall.ps1

```diff
@@ -0,0 +1,13 @@
+$ErrorActionPreference = 'Continue'
+Import-Module vm.common -Force -DisableNameChecking
+
+try {
```

Comment:
Uninstalling is [best effort](https://github.com/mandiant/VM-Packages/wiki/Coding-Conventions#uninstalling-is-best-effort) (that's why we use `$ErrorActionPreference = 'Continue'`). So the `try-catch` is not needed. Please remove it for consistency with other packages.